README.md

```diff
@@ -1,27 +1,31 @@
 # SharpLink
-A lavalink wrapper for Discord.Net!
+A [Lavalink](https://github.com/Frederikam/Lavalink) wrapper for Discord.Net!
 
 [![NuGet Pre Release](https://img.shields.io/nuget/vpre/SharpLink.svg?style=flat-square)](https://www.nuget.org/packages/SharpLink/)
 
-# Getting Started
-Here is some example code to help get you started with SharpLink.
+## `Getting Started`
 
-First you want to make sure you have the LavalinkManager set up.
-```csharp
-DiscordSocketClient client = new DiscordSocketClient();
+- Follow [these instructions](https://github.com/Frederikam/Lavalink#server-configuration) to setup Lavalink.
+- Once Lavalink is up and running follow the code example below to setup Sharplink.
 
-// You must pass the socket client over to the manager for reference
-LavalinkManager lavalinkManager = new LavalinkManager(client, new LavalinkManagerConfig()
+```cs
+var client = new DiscordSocketClient();
+var lavalinkManager = new LavalinkManager(client, new LavalinkManagerConfig
```

Comment:
Any reason for var and not defining type explicitly?

Reply:
I'll need to agree with this and that I prefer using explicit type definitions